ui/src/app/app-routing.module.ts

```diff
@@ -15,6 +16,7 @@ const routes: Routes = [
     { path: 'create-lobby', component: CreateLobbyComponent },
     { path: 'open-lobbies', component: LobbiesListComponent },
     { path: 'lobbies/:lobbyId', component: LobbyComponent },
+    { path: 'open-lobbies/:lobbyId', component: JoinLobbyLinkComponent },
```

Comment:
I thought over naming routes and how it should be done and I think that `lobbies/:lobbyId/join` would be a good option here. If you think different make a comment about it below.